### PR TITLE
feat(discover): Allow more options for the topEvents

### PR DESCRIPTION
**Pull request by wmak**
- This allows up to 10 top events on the discover frontend
- This changes the display *text* to `Top Daily` and `Top Display` and
  adds a new dropdown for the limit
  
### Preview
image

### CODE REVIEW

File: static/app/views/eventsV2/chartFooter.tsx [hunk 1]
```diff
@@ -45,6 +50,10 @@ export default function ChartFooter({
       <SectionValue key="total-value">{total.toLocaleString()}</SectionValue>
     )
   );
+  const topEventOptions: SelectValue<string>[] = [];
+  for (let i = 2; i <= 10; i++) {
```
Comment (wmak) [inline static/app/views/eventsV2/chartFooter.tsx:54]: Not sure if there's an argument for allowing top-1?

**Review by shruthilayaj — APPROVED** [review]
Aside from the dropdown possibly being a bit wonky, LGTM. We will need to update chartcuterie to just the topEvents value from the URL params instead of defaulting to 5 as a follow up too.

**Comment by shruthilayaj** [comment]
Oh and should we add tests?

File: static/app/views/eventsV2/chartFooter.tsx (hunk 1 above)
Comment (wmak) [reply static/app/views/eventsV2/chartFooter.tsx:54]: :+1: Good point, i'll update this to include 1 just so its more aesthetically pleasing

**Pull request merged**